### PR TITLE
feat(container): pnr guide tour mvp

### PR DESCRIPTION
**Pull request by MaximeBajeux**
update guide tour and active on pnr modal

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/pnr-v2` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-13828
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

update guide tour and active on pnr modal

### CODE REVIEW

File: packages/manager/apps/container/src/public/translations/nav-reshuffle/onboarding/Messages_fr_FR.json [hunk 1]
```diff
@@ -1,19 +1,16 @@
 {
   "onboarding_button_text": "C'est parti !",
   "onboarding_introduction_popover_title": "{{userName}}, bienvenue sur votre espace client",
-  "onboarding_introduction_popover_content": "Retrouvez ici toutes les fonctionnalités nécessaires à la gestion de votre compte et de vos services.",
-  "onboarding_introduction_popover_extra": "Votre guide reste disponible et accessible depuis le menu de gauche.",
-  "onboarding_popover_follow_guide_button": "Suivez le guide",
-  "onboarding_popover_hide_button": "Masquer",
+  "onboarding_introduction_popover_content": "Souhaitez-vous voir la visite guidée ?",
+  "onboarding_popover_follow_guide_button": "Voir la visite guidée ",
```
Comment (antonymarion) [inline packages/manager/apps/container/src/public/translations/nav-reshuffle/onboarding/Messages_fr_FR.json:5]: is the trailing space required?

File: packages/manager/apps/container/src/container/nav-reshuffle/sidebar/Assistance.tsx [hunk 2]
```diff
@@ -52,10 +52,10 @@ const AssistanceSidebar: React.FC = (): JSX.Element => {
 
   const trackNode = (id: string) => {
     trackingPlugin.trackClick({ name: `navbar_v2_${id}`, type: 'navigation' });
-  };
+  };
```
Comment (ghyenne) [inline packages/manager/apps/container/src/container/nav-reshuffle/sidebar/Assistance.tsx:55]:
```suggestion
  };
```

File: packages/manager/apps/container/src/public/translations/nav-reshuffle/onboarding/Messages_fr_FR.json [hunk 3]
```diff
@@ -1,19 +1,16 @@
...
-  "onboarding_walkme_popover_step2_title": "Mon profil",
-  "onboarding_walkme_popover_step2_content": "Assurez la gestion de votre compte directement depuis votre profil.",
-  "onboarding_walkme_popover_step3_title": "Mes moyens de paiement",
-  "onboarding_walkme_popover_step3_content": "Ajoutez un ou plusieurs moyens de paiement. Configurez-en un « par défaut » pour vos services en renouvellement automatique.",
-  "onboarding_walkme_popover_step4_title": "Mes services",
-  "onboarding_walkme_popover_step4_content": "Naviguez simplement dans le menu des services proposés par OVHcloud et retrouvez facilement vos services actifs signalés par une pastille.",
-  "onboarding_walkme_popover_step5_title": "Mes factures",
-  "onboarding_walkme_popover_step5_content": "Retrouvez l’ensemble de vos factures, paiements en cours et avoirs dans cette rubrique."
+  "onboarding_walkme_popover_step2_title": "Comment gérer mon compte/mes informations ?",
+  "onboarding_walkme_popover_step2_content": "Mettez à jour votre profil, vos moyens de paiement et votre niveau de support. Accédez à vos factures, vos services et vos commandes. Consultez vos communications et vos contrats. ",
```
Comment (ghyenne) [inline packages/manager/apps/container/src/public/translations/nav-reshuffle/onboarding/Messages_fr_FR.json:11]:
```suggestion
  "onboarding_walkme_popover_step2_content": "Mettez à jour votre profil, vos moyens de paiement et votre niveau de support. Accédez à vos factures, vos services et vos commandes. Consultez vos communications et vos contrats.",
```

**Review by antonymarion — APPROVED** [review]

File: packages/manager/apps/container/src/container/nav-reshuffle/onboarding/introduction.tsx [hunk 4]
```diff
@@ -45,7 +45,7 @@ export const OnboardingIntroduction = () => {
     trackingPlugin.trackClickImpression({
       click: {
         ...commonTrackingOptions,
-        format: `[0-5]`,
+        format: `[0-4]`,
```
Comment (ghyenne) [inline packages/manager/apps/container/src/container/nav-reshuffle/onboarding/introduction.tsx:48]:
```suggestion
        format: '[0-4]',
```

**Review by anooparveti — APPROVED** [review]

**Pull request merged**